src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb

```diff
@@ -50,6 +50,14 @@ Public Module BuildDevDivInsertionFiles
             "codeAnalysisService.servicehub.service.json",
             "remoteHostService.servicehub.service.json",
             "serviceHubSnapshotService.servicehub.service.json",
+            "Microsoft.Build.Conversion.Core.dll",
+            "Microsoft.Build.dll",
+            "Microsoft.Build.Engine.dll",
+            "Microsoft.Build.Framework.dll",
+            "Microsoft.Build.Tasks.Core.dll",
+            "Microsoft.Build.Utilities.Core.dll",
+            "Microsoft.VisualStudio.Threading.dll",
+            "Microsoft.VisualStudio.Validation.dll",
```

Comment:
regardless whether service hub implements a feature for vsix based service hub service to be able to discover dlls defined in devenv.exe.config or not. it would be nice if we can separate MSBuildWorkspace and others which lives in Workspace.Desktop.dll together currently.